### PR TITLE
Local development environment based on Docker

### PR DESCRIPTION
**Pull request by wmalinowski**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.6
+
+COPY . /usr/src/app/
```
Comment (ivellios) [inline Dockerfile]: It is a good practice to copy project files as late in the process as possible. Docker is using caches for builds, but they are based on the previous states, so things that change most often should be placed lower in the file. Also you can copy just requirements first to install libs, and then the whole project separately.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    volumes:
+      - ./:/usr/src/app
+    working_dir: /usr/src/app/meetpy
+    ports:
+      - "8080:8000"
+    tty: true
+    depends_on:
+      - db
+      - smtp
+    environment:
```
Comment (ivellios) [inline docker-compose.yml]: How about using `env_file` for keeping these? They may be reused later in the `smtp` service as well.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+WORKDIR /usr/src/app
+
+RUN pip install --no-cache-dir --requirement requirements/production.txt
+
+ENV DJANGO_SETTINGS_MODULE="meetpy.settings.docker"
+
+EXPOSE 8000
+
+WORKDIR meetpy
+CMD python ./manage.py runserver
```
Comment (ivellios) [inline Dockerfile]: I do not think this command will work. The proper use of CMD is a list of args:
```["python", "manage.py", "runserver"]```

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#cmd

Also you need django to be running dev server on IP 0.0.0.0

So: ```CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]```

Comment (wmalinowski) [reply Dockerfile]: Thanks, it works now

File: docker-compose.yml (hunk 2 above)
Comment (wmalinowski) [reply docker-compose.yml]: Good idea! I'll do it in another PR

**Pull request merged**